ultraplot/axes/geo.py

```diff
@@ -65,6 +73,16 @@
 longridminor, latgridminor, gridminor : bool, default: :rc:`gridminor`
     Whether to draw "minor" longitude and latitude lines.
     Use the keyword `gridminor` to toggle both at once.
+lonticks, latticks : locator-spec, optional
+    Used to determine the longitude and latitude axis tick mark positions. Passed
+    to the `~ultraplot.constructor.Locator` constructor. Can be float,
+    list of float, string, or `matplotlib.ticker.Locator` instance.
+    Use `[]`, `'null'`, or `'none'` for no ticks.
```

Comment:
This needs to lonticklen, latticklen, and ticklen. The extra semantics around Boolean values needs to be removed. Ticklen should set the length for both Lon and lat. The principle here is that the api should be identical.

Reply:
ah I thought you were referring to xtick and ytick